Core/HW/SasAudio.h

```diff
@@ -38,6 +38,7 @@ enum {
 	PSP_SAS_PITCH_MAX = 0x4000,
 
 	PSP_SAS_VOL_MAX = 0x1000,
+	PSP_SAS_MAX_GRAIN = 1024,   // VERY conservative! 256 is quite common but don't think I've ever seen bigger.
```

Comment:
`__sceSasInit` allows `grainSize` of 2048 at most, any higher values fail.  But 2048 is allowed.

-[Unknown]